torch/lib/c10d/comm.hpp

```diff
@@ -57,6 +57,11 @@ class GradBucket {
     return index_ == 0;
   }
 
+  // Overwrites tensors at a specific index.
```

Comment:
I couldn't find test files for this or the init.cpp anywhere -- let me know if I missed it!

Reply:
There isn't any test for init.cpp, as all the C++ functions can be viewed as internal functions. We only need to test python public APIs.